common-lib/src/main/scala/com/gu/mediaservice/model/usage/MediaUsage.scala

```diff
@@ -1,26 +1,34 @@
 package com.gu.mediaservice.model.usage
 
 import org.joda.time.DateTime
+import play.api.Logger
 
 case class UsageId(id: String) {
   override def toString = id
 }
 
 case class MediaUsage(
-                       usageId: UsageId,
-                       grouping: String,
-                       mediaId: String,
-                       usageType: UsageType,
-                       mediaType: String,
-                       status: UsageStatus,
-                       printUsageMetadata: Option[PrintUsageMetadata],
-                       digitalUsageMetadata: Option[DigitalUsageMetadata],
-                       syndicationUsageMetadata: Option[SyndicationUsageMetadata],
-                       frontUsageMetadata: Option[FrontUsageMetadata],
-                       lastModified: DateTime,
-                       dateAdded: Option[DateTime] = None,
-                       dateRemoved: Option[DateTime] = None
-                     ) {
+  usageId: UsageId,
+  grouping: String,
+  mediaId: String,
+  usageType: UsageType,
+  mediaType: String,
+  status: UsageStatus,
+  printUsageMetadata: Option[PrintUsageMetadata],
+  digitalUsageMetadata: Option[DigitalUsageMetadata],
+  syndicationUsageMetadata: Option[SyndicationUsageMetadata],
+  frontUsageMetadata: Option[FrontUsageMetadata],
+  lastModified: DateTime,
+  dateAdded: Option[DateTime] = None,
+  dateRemoved: Option[DateTime] = None
+) {
+
+  def isGridLikeId: Boolean = {
+    // remove events from CAPI that represent images previous to Grid existing
+    Logger.info(s"MediaId $mediaId doesn't look like a Grid image. Ignoring usage $usageId.")
+    !mediaId.startsWith("gu-image-")
```

Comment:
Will the log message here always be appropriate? I may be missing context, but it looks like this will write regardless of whether the image looks like a Grid image.

Reply:
Excellent point @jennygrahamjones ! Have corrected this now.